R/ptype-abbr-full.R

```diff
@@ -13,6 +13,8 @@
 #' characters where possible.
 #'
 #' @param x A vector.
+#' @param check_named Add a prefix for named vectors.
```

Comment:
`prefix_named` and `suffix_shape`?

Reply:
I like `prefix_named` and `suffix_shape` too.